packages/components/statistic/src/index.vue

```diff
@@ -0,0 +1,208 @@
+<!-- eslint-disable @typescript-eslint/no-this-alias -->
+<template>
+  <div>
+    fsa
+    <div class="el-statistic">
+      <div class="head">
+        <slot name="title">
+          <span class="title">
+            {{ title }}
+          </span>
+        </slot>
+      </div>
+      <div class="con">
+        <span class="prefix">
+          <slot name="prefix">
+            {{ prefix }}
+          </slot>
+        </span>
+
+        <span class="number" :style="valueStyle">
+          <slot name="formatter"> {{ disposeValue }}</slot>
+        </span>
+        <span class="suffix">
+          <slot name="suffix">
+            {{ suffix }}
+          </slot>
+        </span>
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
```

Comment:
Please use setup syntax.

Reply:
yes.

Reply:
Thank you. I will adjust it as soon as possible